spec/javascripts/app/utils/countdown-timer_spec.js

```diff
@@ -0,0 +1,48 @@
+const proxyquire = require('proxyquireify')(require);
+
+const spy = sinon.spy();
+const countdownTimer = proxyquire('app/utils/countdown-timer.js', {
+  './ms-formatter': spy,
+  '@noCallThru': true,
+}).default;
+
+const fakeEl = {
+  innerHTML: '',
+};
+
+describe('#countdownTimer', () => {
+  it('does nothing if a HTMLElement is not supplied as the first argument', () => {
+    /* eslint-disable no-unused-expressions */
+    expect(countdownTimer(false)).to.be.falsey;
+    /* eslint-disable no-unused-expressions */
```

Comment:
I think its valid to hold our tests to the same standards as the code itself. Chai specifically aliases all of its methods as properties, which makes eslint think we are calling expressions that don't do anything.

I guess my preference would be to find a way to have eslint ignore chai methods, or use another library like [chai-dirty](https://github.com/prodatakey/dirty-chai) to add the `()` back to chai methods.

If the work outweighs the benefits (or if you don't think we gain anything by linting the tests), I'll disable lint in specs.